frontend/src/static/js/components/test/webstatus-stats-missing-one-impl-chart-panel.test.ts

```diff
@@ -311,15 +359,15 @@ describe('WebstatusStatsMissingOneImplChartPanel', () => {
       apiClientStub.getMissingOneImplementationFeatures,
     ).to.have.been.calledWith(
       'firefox',
-      ['chrome', 'safari'],
+      ['chrome', 'edge', 'safari'],
       new Date('2024-01-01'),
     );
 
     const chartClickEventTwo: ChartSelectPointEvent = new CustomEvent(
       'point-selected',
       {
         detail: {
-          label: 'Chromium',
+          label: 'Edge',
```

Comment:
You replaced the chrome case with edge. But we should still have another case to also assert the arguments to `getMissingOneImplementationFeatures` for a given point-selected event for the Chrome label